src/pages/signIn/config.ts

```diff
@@ -1,20 +1,4 @@
 export const config = [
-  {
-    name: 'firstName',
-    type: 'text',
-    label: 'First name *',
-    isError: true,
-    isRequired: true,
-    placeholder: 'your name',
-    errorText: 'this field must be filled',
-  },
-  {
-    name: 'secondName',
-    type: 'text',
-    label: 'Second name',
-    isRequired: false,
-    placeholder: 'your second name',
-  },
   {
     name: 'email',
     type: 'email',
```

Comment:
In this case i think better use  icon ```*``` to check you key ```isRequired``` and dont use this icon with  ``` label: 'Email *',```